test/services/ContactPrefServiceSpec.scala

```diff
@@ -60,6 +60,16 @@ class ContactPrefServiceSpec extends TestUtil with MockUpdateContactPrefConnecto
 
     }
 
+    "an invalid event type is provided" should {
+
+      "return None" in {
```

Comment:
We're testing permanent bounce which is valid and then we have test for an invalid type. Could we have some tests for temporary bounce and rejected.